lib/core/view_models/login_view_model.dart

```diff
@@ -26,6 +30,12 @@ class LoginViewModel extends BaseViewModel with Validators {
   FocusNode get viewFocusNode => _viewFocusNode;
 
   Future<void> login() async {
+    // Snack bar Sample usage
+    // final alertRequest = ConfirmSnackBarRequest((r) => r
+    //   ..message = "message!"
+    //   ..childtext = "childText!!");
+    // await _snackBarService.showSnackBar(alertRequest);
+
```

Comment:
Uncomment and move this to the settings service notifications toggle to have a demo in the app instead of just the comments.